components/mpas-ocean/src/shared/mpas_ocn_tracer_hmix_redi.F

```diff
@@ -238,14 +238,10 @@ subroutine ocn_tracer_hmix_Redi_tend(meshPool, layerThickness, layerThickEdgeMea
 
             k = minLevelEdgeBot(iEdge)
             kappaRediEdgeVal = 0.25_RKIND*(RediKappaScaling(k, cell1) + RediKappaScaling(k, cell2) + &
-                                           RediKappaScaling(k + 1, cell1) + RediKappaScaling(k + 1, cell2))* &
-                               0.25_RKIND*(RediKappaSfcTaper(k, cell1) + RediKappaSfcTaper(k, cell2) + &
-                                           RediKappaSfcTaper(k + 1, cell1) + RediKappaSfcTaper(k + 1, cell2))
+                                           RediKappaScaling(k + 1, cell1) + RediKappaScaling(k + 1, cell2))!* &
```

Comment:
```suggestion
                                           RediKappaScaling(k + 1, cell1) + RediKappaScaling(k + 1, cell2))
```
Maybe I'm missing something but I don't understand the point of these `!* &` comments and think they can be removed for clarity